### ISSUE TITLE
Possible to restrict to a barcode set?

### ISSUE THREAD

**Issue by nickloman** [rrwick/Porechop#20]
Hi Ryan

I've noticed a few occasions where Porechop has picked both a native barcoding set (i.e. labeled as 'reverse') and also PCR barcoding set (labeled 'forward') both of the same barcode number during adaptor screening.

This then seems to result in bad things happening such as all or many reads getting thrown into the 'None' bin.

Is there a way of disabling certain barcode sets if I know in advance which ones I am using. I.e. I would like it just to look for native barcodes and not PCR barcodes.

Thanks in advance!
Nick

**Comment by khyox** [comment]
Hi Nick. While Ryan answers, have you tried to delete them from [adapters.py](https://github.com/rrwick/Porechop/blob/master/porechop/adapters.py)? This is usually working for us in similar cases. Hope this helps.

**Comment by nickloman** [comment]
Yeah I was thinking about doing that - thank you!

**Comment by rrwick** [comment]
Hi Nick,

Porechop _was_ supposed to be doing just that - choosing either the forward or reverse barcodes (based on whichever had stronger results) and ignoring the other. I tried to reproduce your issue, but couldn't make it happen as you described it. That being said, while looking into it I did rewrite some of the logic, so maybe it works better for you now.

I've just made a new release (v0.2.2), so if the same issue still happens on the new release, maybe I could coax you into sharing a read set which has the problem? If I could replicate it I could surely fix it. Let me know how you go!

Ryan
